deploy/hackathon-docker/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  web:
+    image: "kaiyuanshe/hackathon:latest"
+    build:
+      context: ../../
+      dockerfile: deploy/hackathon-docker/Dockerfile
+    ports:
+      - "9988:80"
```

Comment:
Looks like we need two public ports, for server and client respectively

Reply:
yes, 2 if  they are separated.

Reply:
OHP Server is also publicly accessible

Reply:
more specific, just let me know the map,like:
ports:
      - "9988:80"
      - "aaa:bbb"
     ......
how many, list here

Reply:
15000 for server currently